Week3/homework/6-step3.js

```diff
@@ -3,4 +3,19 @@
 const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
-// add your solution here, or add a comment on how you would tackle this problem
+//add your solution here, or add a comment on how you would tackle this problem
+//print out two dimension array
+for (let i = 0; i < arr2d.length; i++) {
+  for (let j = 0; j < arr2d[i].length; j++) {
+    console.log(arr2d[i][j]);
+  }
+}
+
+// print out three dimension array
+for (let i = 0; i < arr3d.length; i++) {
+  for (let j = 0; j < arr2d[i].length; j++) {
+    for (let x = 0; x < arr3d[i][j].length; x++) {
```

Comment:
It is customary when using nested loops likes this to use consecutive letters from the alphabet, starting with the letter `i`, e.g. `i`, `j`, 'k` etc. So in this case, I would use `k` instead of `x`.